projects/packages/paypal-payments/package.json

```diff
@@ -15,11 +15,48 @@
 	"license": "GPL-2.0-or-later",
 	"author": "Automattic",
 	"scripts": {
-		"build": "echo 'Not implemented.'",
+		"build": "pnpm run clean && webpack --config ./webpack.config.blocks.js",
 		"build-js": "echo 'Not implemented.'",
 		"build-production": "echo 'Not implemented.'",
 		"build-production-js": "echo 'Not implemented.'",
```

Comment:
You can drop `build-js` and `build-production-js` if you're not going to use them. OTOH, you are calling `build-production` from composer.json, so that one should be implemented. It might be as simple as
```suggestion
		"build-production": "NODE_ENV=production BABEL_ENV=production pnpm build",
```

Reply:
Addressed in [0735334](https://github.com/Automattic/jetpack/pull/43413/commits/07353346486079ae3752e3b1af07481526b4629f).